tinkerbell/stack/templates/nginx.yaml

```diff
@@ -183,12 +183,39 @@ spec:
   - port: {{ .Values.smee.syslog.port }}
     protocol: UDP
     name: {{ .Values.smee.syslog.name }}
+{{- if .Values.stack.relay.enabled }}
   - port: 67
     protocol: UDP
     name: dhcp
+{{- end }}
   selector:
     {{- with .Values.stack.selector }}
     {{- toYaml . | nindent 4 }}
     {{- end }}
+{{- if not .Values.stack.relay.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ .Values.smee.name }}
+  name: {{ .Values.smee.name }}-lb
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  type: {{ .Values.stack.service.type }}
+  {{- if eq .Values.stack.service.type "LoadBalancer" }}
+  {{- if .Values.stack.lbClass }}
+  loadBalancerClass: {{ .Values.stack.lbClass }}
+  {{- end }}
+  loadBalancerIP: {{ .Values.stack.loadBalancerIP }}
```

Comment:
this looks to be the same value as the stack service object. I dont think 1 load balancer IP for 2 services is going to work.

Reply:
I already tried and it can work as the ports are different. But I was considering whether we should use hostNetwork for smee in this situation or not?

Reply:
@jacobweinstock Can you take a look again? I guess this one makes more sense.